Header header
float64 pick_object_X
float64 pick_object_Y
float64 place_object_X
float64 place_object_Y

